detector/aws_db_instance_invalid_parameter_group.go

```diff
@@ -9,22 +9,20 @@ import (
 
 type AwsDBInstanceInvalidParameterGroupDetector struct {
 	*Detector
-	IssueType       string
-	TargetType      string
-	Target          string
-	DeepCheck       bool
 	parameterGroups map[string]bool
 }
 
 func (d *Detector) CreateAwsDBInstanceInvalidParameterGroupDetector() *AwsDBInstanceInvalidParameterGroupDetector {
-	return &AwsDBInstanceInvalidParameterGroupDetector{
+	nd := &AwsDBInstanceInvalidParameterGroupDetector{
 		Detector:        d,
-		IssueType:       issue.ERROR,
-		TargetType:      "resource",
-		Target:          "aws_db_instance",
-		DeepCheck:       true,
 		parameterGroups: map[string]bool{},
 	}
+	nd.Name = "aws_db_instance_invalid_parameter_group"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_db_instance"
```

Comment:
__[gometalinter]__
* [goconst] 8 other occurrence(s) of "aws_db_instance" found in: aws_db_instance_invalid_type.go:23:14 aws_db_instance_readable_password.go:17:14 aws_db_instance_invalid_option_group.go:23:14 aws_db_instance_invalid_vpc_security_group.go:24:14 aws_db_instance_default_parameter_group.go:20:14 aws_db_instance_previous_type.go:23:14 aws_db_instance_duplicate_identifier.go:23:14 aws_db_instance_invalid_db_subnet_group.go:23:14

<!-- Comment by [SideCI](https://sideci.com) -->